README.md

```diff
@@ -104,6 +113,16 @@ See [HACKING.md](HACKING.md) for further instructions
 
 ## Command interface
 
+### Shell Keys
+- Enter
+ - Runs the statement
+- Escape+Enter
+ - Starts a new line to enter a statement that spans multiple lines. You can also copy input text with multiple lines and paste it into the shell. For instructions on setting up Option instead of Escape as a Meta key in macOS, see the [OS X Daily](https://osxdaily.com/2013/02/01/use-option-as-meta-key-in-mac-os-x-terminal/) site. 
+- Ctrl+C
+ - Noop
```

Comment:
The **Ctrl+C** will cancel the current command, which looks like:
```
qldb> select * from
CTRL-C
qldb>
```

Reply:
ah, thanks for clarifying. i was wondering about this, because i see some shells doing it differently. in the alpha13 release it exited the shell immediately. however, i prefer the current behavior, where it kills the ongoing command but doesn't kill the entire shell. will update.